platforms/shared/configuration/kubernetes-env-setup.yaml

```diff
@@ -46,6 +46,9 @@
       kubeconfig_path: "{{ item.k8s.config_file }}"
       kubecontext: "{{ item.k8s.context }}"
       aws: "{{ item.aws }}"
+      email: "{{ item.gitops.email }}"
+      organization: "{{ item }}"
+      component_ns: "{{ item.name | lower }}-bes"
```

Comment:
the component_ns variable should not be suffixed with bes here

Reply:
corrected